permissions/plugin-github-branch-pr-change-filter.yml

```
@@ -0,0 +1,7 @@
+---
+name: "github-branch-pr-change-filter"
+github: "jenkinsci/github-branch-pr-change-filter-plugin"
+paths:
+- "org/jenkins-ci/plugins/github-branch-pr-change-filter"
```

```suggestion
- "io/jenkins/plugins/github-branch-pr-change-filter"
```

See https://github.com/jenkinsci/github-branch-pr-change-filter-plugin/blob/master/pom.xml#L10

---

Thank you! I was looking at the wrong group id, the one for the parent POM